libs/blocks/preflight/panels/performance.js

```diff
@@ -144,11 +144,17 @@ export const checkForPersonalization = () => conditionalItemUpdate({
   key: text.personalization.key,
 });
 
-export const checkVideosWithoutPosterAttribute = () => conditionalItemUpdate({
-  failsWhen: !config.lcp.element.poster,
-  emptyWhen: !config.lcp.url.match('media_.*.mp4'),
-  key: text.videoPoster.key,
-});
+export const checkVideosWithoutPosterAttribute = () => {
+  const hasVideoUrl = config.lcp.url.match('media_.*.mp4');
+  const lcpElement = config.lcp.element;
+  const videoElement = lcpElement?.tagName === 'VIDEO' ? lcpElement : lcpElement?.closest('video') || lcpElement?.querySelector?.('video') || document.querySelector('main > div.section video');
```

Comment:
Does it really need so many different checks?
I'd imagine it only needs `config.lcp.element.tagName === 'VIDEO' || config.lcp.element.querySelector.('video')` - as far as I know, this will also fail early if there's no LCP, so you don't need the extra `?.` 

When I developed this, I skipped on checking `document.querySelector('main > div.section video');` given that this might not be the LCP element. We should stick true to what google actually discovers as LCP, if it doesn't there might be other issues with far worse performance impact.
Though, feel free to challenge this assumption, it's been a while since working on this